src/test/java/com/hocs/test/pages/create_case/CreateCase.java

```diff
@@ -97,31 +112,70 @@ public void enterCaseDetailsFreeText() {
         caseDetailsFreeTextField.sendKeys(generateRandomString());
     }
 
+    public void fillMandatoryDateFields(){
+        enterDispatchDeadlineDay(todayPlusNDaysGetDay(+365));
+        enterDispatchDeadlineMonth(todayPlusNDaysGetMonth(+365));
+        enterDispatchDeadlineYear(todayPlusNDaysGetYear(+365));
+        enterDraftDeadlineDay(todayPlusNDaysGetDay(+360));
+        enterDraftDeadlineMonth(todayPlusNDaysGetMonth(+360));
+        enterDraftDeadlineYear(todayPlusNDaysGetYear(+360));
+    }
+
+    private void enterDispatchDeadlineDay(String day){
```

Comment:
maybe you could move these all into one method 
enterDispayedDeadlineDate(string, day, string month, string year); 

?